projects/ion/src/lib/picker/date-picker/date-picker-calendar/date-picker-calendar.component.ts

```diff
@@ -4,68 +4,67 @@ import {
   OnInit,
   Output,
   EventEmitter,
-  AfterViewInit,
+  DoCheck,
 } from '@angular/core';
-import { SafeAny } from '../../utils/safe-any';
-import { Calendar } from '../core/calendar';
-import { Day } from '../core/day';
+import { SafeAny } from '../../../utils/safe-any';
+import { Calendar } from '../../core/calendar';
+import { Day } from '../../core/day';
 
 type DateEmitter = {
-  date: string;
+  day: Day;
 };
 
-type DateField = {
-  date: Day;
-  label: string;
+export type UpdateLabelCalendar = {
+  month: string;
```

Comment:
If you created a Month type like I said above, you could use it here.

```suggestion
  month: Month;
```